### PR TITLE
feat(timeformat): Replace `date-fns` with a simple time format

### PR DESCRIPTION
**Pull request by tomerlichtash**

### CODE REVIEW

File: src/components/ui/time-formart/time-format-consts.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export const LocaleTimeFormats = {
```
Comment (imdfl) [inline src/components/ui/time-formart/time-format-consts.ts:1]: I think that all the locale data should be in one file that defines a locale "bundle".

**Review by imdfl — APPROVED** [review]
It's ok w me if we postpone locale bundling, but I think it should be done.

File: src/components/ui/time-formart/time-format-consts.ts (hunk 1 above)
Comment (tomerlichtash) [reply src/components/ui/time-formart/time-format-consts.ts:1]: I agree. Will push an update.

Comment (tomerlichtash) [reply src/components/ui/time-formart/time-format-consts.ts:1]: @imdfl Do you think this data should in the `LocaleInfo` object? e.g.

```
export const LocaleInfo: Record<LocaleId, ILocaleInfo> = {
	en: {
		direction: "ltr",
		timestamp: {
			format: "MM-dd-yyyy",
			delim: "-"
		}
	},
.
.
.
```

**Pull request closed without merging**